airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityImpl.java

```diff
@@ -80,6 +86,15 @@ public GeneratedJobInput getSyncWorkflowInput(final SyncInput input) {
 
       final JobRunConfig jobRunConfig = TemporalWorkflowUtils.createJobRunConfig(jobId, attempt);
 
+      List<StandardDestinationDefinition> destinationDefinitionList = configRepository.listStandardDestinationDefinitions(true);
+      Optional<StandardDestinationDefinition> optionalDestinationDefinition = destinationDefinitionList.stream()
+          .filter(destinationDefinition -> config.getDestinationDockerImage()
+              .equalsIgnoreCase(destinationDefinition.getDockerRepository() + ":" + destinationDefinition.getDockerImageTag()))
```

Comment:
note: there's a utility function `DockerUtils.getTaggedImageName` that deals with combining the repo and tag. Not the most complicated thing to do here but might as well use it if it's available.

Reply:
I have not yet come across the use of this extra class. Thanks for sharing it)